client/components/illustration/index.jsx

```diff
@@ -0,0 +1,12 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const Illustration = ( { illustration } ) => (
+	<div className="illustration-container">
```

Comment:
I was wondering that. I had found [this example](https://github.com/Automattic/wp-calypso/blob/master/client/components/empty-content/index.jsx) of similar usage, but digging deeper, I can see it was done this way dating back from [before public release](https://github.com/Automattic/wp-calypso/blame/b5028b8f60a5dc852b3651498fadbb4fdaf2dbc4/client/components/empty-content/empty-content.jsx#L101). I'll get rid of the component.